docs/UserGuide.md

```diff
@@ -540,16 +569,20 @@ Examples: <br>
 <div markdown="span" class="alert alert-warning">:exclamation: **Constraints:**<br>
 
 * Name and Note are compulsory fields.<br>
-* Name is case-insensitive but space-sensitive <br>
-* Name must be present in Pooch Planner.<br>
-* Note cannot be empty 
+* Deadlines are optional fields. <br>
+* Deadlines must follow "YYYY-MM-DD" format. <br>
+* Name is case-insensitive but space-sensitive. <br>
```

Comment:
Change "name" to plural form to standardise with "deadlines",

i.e. "Names are case-insensitive but space-sensitive."

Same for the following 2 lines

Reply:
changed